frontend/src/metabase/icon_paths.ts

```diff
@@ -34,11 +34,9 @@ export const ICON_PATHS: Record<string, any> = {
     "M30.595 13.536c1.85.755 1.879 2.05.053 2.9l-11.377 5.287c-1.82.846-4.763.858-6.583.022L1.344 16.532c-1.815-.835-1.785-2.131.05-2.89l1.637-.677 8.977 4.125c2.194 1.009 5.74.994 7.934-.026l9.022-4.193 1.63.665zm-1.63 7.684l1.63.666c1.85.755 1.879 2.05.053 2.898l-11.377 5.288c-1.82.847-4.763.859-6.583.022L1.344 24.881c-1.815-.834-1.785-2.131.05-2.89l1.637-.677 8.977 4.126c2.194 1.008 5.74.993 7.934-.026l9.022-4.194zM12.686 1.576c1.843-.762 4.834-.77 6.687-.013l11.22 4.578c1.85.755 1.88 2.05.054 2.899l-11.377 5.288c-1.82.846-4.763.858-6.583.022L1.344 9.136c-1.815-.834-1.785-2.13.05-2.89l11.293-4.67z",
   archive: {
     path:
-      "M14 2.1875H0V4.74189H1.21739H12.7826H14V2.1875ZM12.7826 5.38048H1.21739V11.7664H12.7826V5.38048ZM4.26087 6.65768H9.73913V7.29627H4.26087V6.65768Z",
+      "M6.857 2.762c0-.42.341-.762.762-.762h16.762a.762.762 0 0 1 0 1.524H7.619a.762.762 0 0 1-.762-.762zM0 11.809a6 6 0 0 1 6-6h20a6 6 0 0 1 6 6v12.382a6 6 0 0 1-6 6H6a6 6 0 0 1-6-6V11.81zm10.62 3.097a1.143 1.143 0 0 1 1.617 0L16 18.67l3.763-3.763a1.143 1.143 0 0 1 1.617 1.616l-4.572 4.572a1.143 1.143 0 0 1-1.616 0l-4.572-4.572a1.143 1.143 0 0 1 0-1.616z",
```

Comment:
So, this seems to be a revert to the state prior to #23315.

But I can't wrap my head around the rest of the icons. Most of them (if not all) are some new (third) version/iteration, if I'm reading it correctly?

Reply:
They're the same icons as before, but their paths have been updated to render as close to 32 x 32 as they can (keeping aspect ratio). The breaking change before was actually that I was specifying a viewBox to compensate for smaller SVGs, but in many places in the app, we expect a standard size of 32x32